src/dodal/devices/undulator.py

```diff
@@ -23,6 +53,7 @@ class Undulator(StandardReadable):
     def __init__(
         self,
         prefix: str,
+        id_gap_lookup_table_path: str,
```

Comment:
i22 undulator still doesn't have this required constructor arg

Reply:
fixed